src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -7814,7 +7787,7 @@ private Symbol VisitMemberAccess(BoundExpression node, BoundExpression receiverO
             SpecialMember? nullableOfTMember = null;
             if (member.RequiresInstanceReceiver())
             {
-                member = AsMemberOfType(receiverType.Type, member);
+                member = AsMemberOfType(receiverType.Type!, member);
```

Comment:
How do we know that `Type` isn't null?

Reply:
We don't. Good catch.

---
In reply to: [463188369](https://github.com/dotnet/roslyn/pull/45640#discussion_r463188369) [](ancestors = 463188369)

Reply:
assertion?